DOCS-10220-create-and-implement-nav-partial (#75)
mulesoft/docs-visualizer: modules/ROOT/pages/view.adoc

include::_attributes.adoc[]
endif::[]

> Users who have the Visualizer Editor permission can create auto-discoverable views that provide a single source of truth and that are accessible to their team members across environments.

Reviewer: ```suggestion
Users with the Visualizer Editor permission can create auto-discoverable views that provide a single source of truth and are accessible to their team members across environments.
```

* New services deployed that match the selected environments and tags
* If the *Show dependents and dependencies* option is selected and the a service is deployed and related to a selected service

> Views that you have access to appear in a list in the *View* card (click the drop-down arrow to see the list). This gives you easy access to all the views that are available to you, and saves you time because you don't need to readjust filters every time you access Anypoint Visualizer to get the view that is relevant to you.

Reviewer: ```suggestion
For quick access and to save you from readjusting the filters every time you use Anypoint Visualizer, all the views to which you have access appear in a list in the *View* card (click the drop-down arrow to see the list). 
```

Author: Written as it was to emphasize the UI object before anything else.

== Defined Views

> If you have the Visualizer Editor permission, you can create views that provide a consistent, high quality source of truth for the architecture of each of your organization's projects.

Reviewer: ```suggestion
If you have the Visualizer Editor permission, you can create views that provide a consistent and high-quality source of truth for the architecture of each of your organization's projects.
```
